examples/cvae/main.py

```diff
@@ -0,0 +1,105 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import pandas as pd
+import pyro
+import torch
+import baseline
+import cvae
+from util import get_data, visualize, generate_table
+
+
+def main(args):
+    device = torch.device("cuda:0" if torch.cuda.is_available() and args.cuda
+                          else "cpu")
+    results = []
+    columns = []
+
+    for num_quadrant_inputs in args.num_quadrant_inputs:  # [1, 2, 3]
+        maybes = 's' if num_quadrant_inputs > 1 else ''
```

Comment:
what's `maybes` doing? could this use some comments?

Reply:
This just make sure the final table has headers ['1 quadrant', '2 quadrants', '3 quadrants'] (the first is singular, the 2nd and 3rd are plural). But I added a comment clarifying..